terraform/modules/01-inputs-required.tf

```diff
@@ -0,0 +1,11 @@
+variable "environment" {
```

Comment:
drop numbers in new files, 01 etc. it's an anti-pattern that does nothing other than confuse

Reply:
sure - I guess the reason why its done to keep the files in the order you want to see it but happy to drop them :)

Reply:
make sense